### ISSUE TITLE
Cannot package for "linux x64 or "linux ia32".

### ISSUE THREAD

**Issue by JamenMarz** [maxogden/electron-packager#135]
I'm using Linux Mint 17.1, and I receive these errors when trying to package either "linux x64" or "linux ia32" with Electron v0.32.3.  As the error states, appears to be some sort of directory error.

<s>When I downloaded the "linux x63" binaries form Electron's repo yesterday, I don't believe there was a `resources/app` folder in it.  Perhaps this is the cause of the issue.</s>

Trying for Linux x64:
```
$ sudo electron-packager ./ FooBar --platform=linux --arch=x64 --version=0.32.3
Packaging app for platform linux x64 using electron v0.32.3
[ { [Error: ENOENT: no such file or directory, mkdir '/tmp/electron-packager/linux-x64/FooBar-linux-x64/resources/app']
    errno: -2,
    code: 'ENOENT',
    syscall: 'mkdir',
    path: '/tmp/electron-packager/linux-x64/FooBar-linux-x64/resources/app' } ] undefined
```

Trying for Linux x32:
```
$ sudo electron-packager ./ FooBar --platform=linux --arch=ia32 --version=0.32.3
Downloading electron-v0.32.3-linux-ia32.zip
[============================================>] 100.0% of 36.81 MB (121.07 kB/s)
Packaging app for platform linux ia32 using electron v0.32.3
[ { [Error: ENOENT: no such file or directory, mkdir '/tmp/electron-packager/linux-ia32/FooBar-linux-ia32/resources/app']
    errno: -2,
    code: 'ENOENT',
    syscall: 'mkdir',
    path: '/tmp/electron-packager/linux-ia32/FooBar-linux-ia32/resources/app' } ] undefined

```

**Comment by malept** [comment]
electron-packager depends on the extract-zip package, which should have fixed this bug this morning (PDT). Which version of extract-zip is installed? The affected version is 1.1.0, the fixed version is 1.1.1.

**Comment by JamenMarz** [comment]
Yup, appears to work in the fixed version. :)
